content/en/blog/sc-q1-report.md

```diff
@@ -0,0 +1,65 @@
+
+> 🔗 View board: [https://github.com/orgs/todogroup/projects/9](https://github.com/orgs/todogroup/projects/9)
+
+### 2025 Strategic Goals:
+
+1. **Develop Shared Goals with Partner Communities**  
+2. **Establish Touchpoint Committee & Plan in Advance Sessions with Speakers**  
+3. **Recognize TODO Contributors and Build Processes**  
+4. **TODO Repo Maintenance and Clean-Up**  
+
+## Q1 Progress Highlights
+
+📘 **Ambassador Guidelines Updated**: Revised and published with clearer [onboarding and engagement expectations](https://github.com/todogroup/governance/blob/main/TODO-OSPO-Ambassador-Program.md).
+
+🗓️ **Touchpoint Planning Agenda Created** A structured [agenda](https://github.com/todogroup/ospology/tree/main/meetings/Agenda_2025#todo-touchpoint-agenda-overview-january---june-2025) now invites members to: Share past conference talks; Introduce new tools/initiatives; Propose hot topics for discussion; Join open forums under Chatham House Rule  
+
+> This new structure brought a 114% increase in participation compared to 2024 Q1.
+
+🎓 **First TODO Certification Launched**: In collaboration with the Open Source Initiative and LF Education, we launched the first TODO certification:[Certified Open Source Developer for Enterprise Code](https://training.linuxfoundation.org/certification/certified-open-source-developer-for-enterprise-code/)
+
+🔄 **Steering Committee Update**: One of the SC members, **Nik Peters**, has stepped down from their role. We’re grateful for Nik’s contributions to TODO, including sharing Porsche’s OSPO journey in the OSPO Book and participating in the TODO 10th Anniversary video. Following the results of the last SC elections, **Alice Sowerby** will fill the vacant seat through December 2025. Alice is one of the OSPO Book chairs, a highly active TODO contributor, and contributor to the CHAOSS podcast. **Welcome, Alice!**
+
+🔐 **Closer Connection with Security**: Representatives from Open Source Security Foundation (OpenSSF) actively participated in **OSPOlogyLive Amsterdam**, sharing open source management guidance for security teams to open source managers. Moreover, a full **Security Chapter** (Chapter 5) has been featured in the OSPO Book as a result of this collaboration.
+
+🤝 **Sustain Partnerships to Integrate Standards and Specifications into OSPO Practices**: TODO supported and participated in the [OpenChain and Friends](https://openchainproject.org/) held in Stuttgart in April.
+
+> While TODO provides **methodologies** (guidance on integrating open source practices and engagements into organizational workflows) for strategic open source management, the OpenChain Project offers **standards frameworks** to ensure that open source compliance and security meet industry benchmarks across the software supply chain. Together, they form two complementary mechanisms that guide organizations through the complexities of open source usage and contribution.
```

Comment:
```suggestion
> While TODO focuses on policy and process choices for effectively open source engagements within organizations, OpenChain provides process management standards, alongside practical guides, to ensure trusted and consistent open source compliance across the supply chain. Together, the two Linux Foundation projects offer complementary approaches to strategic open source management, with both communities regularly exchanging information and resources
```

Reply:
Changed this based on Shane's input to make sure it reflects OpenChain's current activities. Needs to review before merging